### ISSUE TITLE
Exporting to Microsoft project

### ISSUE THREAD

**Issue by CGibsonZor** [bardsoftware/ganttproject#1385]
Error message when trying to export to Microsoft Project
[Ganttproject Export to MS Project Error Message.pdf](https://github.com/bardsoftware/ganttproject/files/889039/Ganttproject.Export.to.MS.Project.Error.Message.pdf)

**Comment by dbarashev** [comment]
Dear Colin.
When you get such error message, you see "View log" link in its text. Please click the link and copy all text from the dialog which will show up. 

And it would be nice to see the project file and to know which format you're trying to export to -- MPX or MSPDI

Thanks.

**Comment by CGibsonZor** [comment]
Dear Dmitry,

 

1.       I clicked the link and there were pages and pages of text – which I could not copy – presumably it was too much.

2.       I had tried to attach the project file to the Issue Tracker but there was yet another error statement advising that that type of file was not supported – s o I had to do a screen shot and send a pdf (which was an acceptable file type listed)

3.       I was trying to export to Microsoft project.  In the dialogue box there are two choices and MS project is given only as MSPDI.

 

As I can attach the project file to this email I am doing so.  It can be opened, export tried, and the error log file accessed I presume.

 

Thanks

 

Col Gibson

**Comment by dbarashev** [comment]
Sorry to disappoint you, but the issue tracker on GitHub is pretty restrictive, and doesn't support attachments in email replies.

It supports ZIP attachments though, so you can ZIP your project file and logs and attach them in the web interface.

Yes, we need those pages and pages of text. They all sit in `ganttproject.log` file in your home directory. Send them to us.

**Comment by CGibsonZor** [comment]
Dear Dmitry,

 

I am a structural engineer and so really do not know what all this is about!  I certainly do not want to get heavily involved in GitHub and all the tabs etc that I saw displayed.  I am not trying to help write a programme – just sue it!

 

1.       Why is there a need to Zip and upload the project file when I sent it to you by email.

2.       All the pages of text can be seen when you, or whoever, looks at the error log when using my project file to try to export.  I mentioned that I could not copy the pages so I cannot see how they can be sent.

 

Regards

 

Colin Gibson

**Comment by CGibsonZor** [comment]
Hi again,

 

I tried again with the project file and this time there was less text – which I have copied and pasted below.  I hope this is what you needed.

 

R

 

Colin G

 

 

 

C:\Users\User\ganttproject.log

 

GanttProject 2.8.4 Pilsen (build 2134)

Settings file:

                location: C:\Users\User\.ganttproject

                size:5075

                is readable: true

java.class.path: eclipsito.jar;C:\Program Files\GanttProject-2.8

java.home: C:\Program Files\Java\jre1.8.0_121

java.ext.dirs: C:\Program Files\Java\jre1.8.0_121\lib\ext;C:\Windows\Sun\Java\lib\ext

java.io.tmpdir: C:\Users\User\AppData\Local\Temp\

java.runtime.version: 1.8.0_121-b13

java.vendor: Oracle Corporation

java.vm.name: Java HotSpot(TM) Client VM

java.vm.vendor: Oracle Corporation

java.vm.version: 25.121-b13

os.arch: x86

os.name: Windows 7

os.version: 6.1

sun.java.command: org.bardsoftware.eclipsito.Boot ganttproject-eclipsito-config.xml -log true D:\Current Work\Projects\Hubco\Outlet Channel\Programme\Hub Outlet Channel - Rev 3 - Mar '17.gan

user.country: US

user.dir: C:\Program Files\GanttProject-2.8

user.home: C:\Users\User

user.language: en

user.timezone: Asia/Karachi

 

Creating main frame...

1. loading look'n'feels

no editor for column=0 class=interface javax.swing.Icon

no editor for column=1 class=interface javax.swing.Icon

no editor for column=2 class=interface javax.swing.Icon

2. loading options

3. creating menus...

4. creating views...

5. calculating size and packing...

6. changing language ...

7. first attempt to restore bounds

8. finalizing...

Main frame created

no editor for column=0 class=interface javax.swing.Icon

no editor for column=1 class=interface javax.swing.Icon

no editor for column=2 class=interface javax.swing.Icon

Splash closed

Resizing window...

no editor for column=0 class=interface javax.swing.Icon

no editor for column=1 class=interface javax.swing.Icon

no editor for column=2 class=interface javax.swing.Icon
